src/components/app/AppViewRouter.js

```diff
@@ -81,7 +82,7 @@ class AppViewRouterImpl extends PureComponent<AppViewRouterProps> {
         return <ProfileLoaderAnimation />;
       case 'FATAL_ERROR': {
         const message =
-          ERROR_MESSAGES[dataSource] || "Couldn't retrieve the profile.";
+          ERROR_MESSAGES[dataSource] || 'AppViewRouter--error-message-compare';
```

Comment:
super nit: this is probably better to use `AppViewRouter--error-message-public` as fallback value.